File: minesweeper-api/src/main/java/com/deviget/minesweeperapi/model/responseDto/CellResponse.java
package com.deviget.minesweeperapi.model.responseDto;

import lombok.Data;

@Data
public class CellResponse {
    private long id;
    private int xPosition;
    private int yPosition;
    private Integer value;
    private Boolean mine;
    private Boolean flag;
    private Boolean revealed;

}

File: MinesWeeper.APIClientLibrary/MinesWeeper.APIClientLibrary/Model/CellResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MinesWeeper.APIClientLibrary.Model
{
    public class CellResponse
    {
        public long id { set; get; }
        public int xPosition { set; get; }
        public int yPosition { set; get; }
        public int value { set; get; }
        public bool mine { set; get; }
        public bool flag { set; get; }
        public bool revealed { set; get; }
    }
}
